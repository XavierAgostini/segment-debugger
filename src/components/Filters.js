import React from 'react'
import { Pane, Radio, SearchInput, Tooltip, Button, FilterCheckbox , Icon} from 'evergreen-ui'

export default class Filters extends React.Component {
  render() {
    return (
      <div>
        <div>

          <div>
            <Pane>Search</Pane>
            <Tooltip
              content="By default, the search pulls results only from the Event Property Values. To include Property Keys in search results, select Full Event Payload."
            >
              <Icon icon="user"></Icon>
            </Tooltip>
          </div>

          <div>
            <Pane display="flex">
              <Radio
                label="Event Property Values"
                name={`${this.htmlId}-search`}
                value="false"
                size={16}
                marginY="0"
              />
              <Radio
                label="Full Event Payload"
                name={`${this.htmlId}-search`}
                value="true"
                size={16}
                marginY="0"
                marginLeft="10"
              />
            </Pane>
          </div>

        </div>
      </div>
    )
  }
}