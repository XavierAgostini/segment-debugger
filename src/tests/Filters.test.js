import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Filters from '../components/Filters';

test('should render DebuggerApp correctly', () => {
  const renderer = new ReactShallowRenderer()
  renderer.render(<Filters/>)
  expect(renderer.getRenderOutput()).toMatchSnapshot()
})
