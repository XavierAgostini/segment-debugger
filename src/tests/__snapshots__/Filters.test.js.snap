// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DebuggerApp correctly 1`] = `
<div>
  <div>
    <div>
      <_class>
        Search
      </_class>
      <Tooltip
        appearance="default"
        content="By default, the search pulls results only from the Event Property Values. To include Property Keys in search results, select Full Event Payload."
        hideDelay={120}
        position="bottom"
      >
        <_class
          icon="user"
        />
      </Tooltip>
    </div>
    <div>
      <_class
        display="flex"
      >
        <_class
          label="Event Property Values"
          marginY="0"
          name="undefined-search"
          size={16}
          value="false"
        />
        <_class
          label="Full Event Payload"
          marginLeft="10"
          marginY="0"
          name="undefined-search"
          size={16}
          value="true"
        />
      </_class>
    </div>
  </div>
</div>
`;
